src/LRSResponse.php

```diff
@@ -30,4 +30,16 @@ public function __construct($success, $content, $httpResponse) {
         $this->content = $content;
         $this->httpResponse = $httpResponse;
     }
+
+    public static function fromRemoteLRSResponse(array $response, array $options, $content) {
```

Comment:
Haven't used type hinting anywhere else, is there a reason to add it here? Something in PHP7 that requires it in the few locations it is included?